src/Sarif.Driver/Sdk/SkimmerBase.cs

```diff
@@ -82,8 +82,6 @@ internal static string FirstSentence(string fullDescription)
 
         public IDictionary<string, string> Options { get; }
```

Comment:
Interesting post v1 thought: express tool parameterization via an options JSON blob that's handled with the same machinery as the properties. Your code could replace/provide an alternate for the current approach (which uses XML serialization). This would also allow us to express this parameterization directly in a SARIF log file.